js-refresher2/index.js

```diff
@@ -0,0 +1,25 @@
+const firstName = "Joshua";
+const lastName = "Propst";
+const fullName = firstName.concat(" " + lastName);
+console.log(fullName);
+
+let population = 1625000;
+console.log(population);
+let total = 1625000 * 3;
+console.log(total)
+
+const javaScriptIsCool = true;
+
+const color1 = "red";
+const color2 = "orange";
+const color3 = "yellow";
+const color4 = "green";
+const color5 = "blue";
+const color6 = "indigo";
+const color7 = "violet";
+const colors = color1.concat(" ", color2, " ", color3, " ", color4, " ", color5, " ", color6, " ", color7);
```

Comment:
You joined each of the color strings into one long string. The build specs were asking for an array containing these strings. You could do this a few different ways, but these are the two easiest: 

const colors = ["red", "orange", "yellow", "green", "blue", "indigo", "violet"];
or because you already asigned the strings to variables, we could use the variable names:
const colors = [ color1, color2, color3, color4, color5, color6, color7 ];